### DISCUSSION TITLE
Should this forum be public?

### DISCUSSION THREAD

**Discussion by jmarshrossney** [ukceh-rse/community#5] [Polls]
This could be an interesting poll if we were able to share it more widely!

**Public:** Anyone with an internet connection can view, and any GitHub user can contribute.

**Private to NERC-CEH:** Only members of the NERC-CEH organisation, and invited collaborators, will be able to view and contribute. [Collaborators on private repositories cost money](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization) (assuming we are on a paid plan).

**Private to a new organisation:** Another possible option if we really wanted to try to bring in friends and colleagues from outside UKCEH.

**Comment by mattjbr123** [comment]
When I was doing my PhD/Masters I benefitted a lot from discussion/help-me forums like this, so my vote is for public!

**Comment by jmarshrossney** [comment]
Based on feedback from the Edinburgh Data Group meeting today, there seems to be a desire for an internal-only discussion forum, and concern about the public model.

People seemed to want somewhere they could discuss issues pertaining to closed-source code, or code not owned by UKCEH. They were not sold on the idea that they could have more generic discussions on a public forum, keeping the details private or linking to a private repository.

I take some of this with a pinch of salt; the cultural factors are being underplayed. But also we need something to serve the needs of UKCEH scientists *today* and a lot of that is of the form "how do I do X using Datalabs, or Jasmin, or a CEH virtual machine".

In future we can think about a more open knowledge-sharing forum that is accessible to aligned organisations, ex-CEH people etc.

So I think I might vote for _private_.

I will also add some more options which involve keeping this one, but making a new private one. EDIT: Damn that resets the votes!
